tf2onnx/onnx_opset/logical.py

```diff
@@ -121,3 +121,17 @@ def version_7(cls, ctx, node, **kwargs):
         new_node = ctx.insert_new_node_on_output("Not", output_name, name=utils.make_name(node.name))
         ctx.copy_shape(output_name, new_node.output[0])
         ctx.set_dtype(new_node.output[0], ctx.get_dtype(output_name))
+
+
+@tf_op("GreaterEqual", onnx_op="GreaterOrEqual")
+class GreaterEqual:
```

Comment:
so how this this work for opset < 12 ?
If the registration at the old place works it is accidental ... you should move the pre opset 11 code to here was well.

Reply:
Added case covering older version.

Reply:
dup removed.